src/components/PlantsV2/components/TotalReportedPlantsCard.tsx

```diff
@@ -50,7 +55,7 @@ export default function TotalReportedPlantsCard({ plantingSiteId }: TotalReporte
           </Typography>
           <Box display='flex' alignItems='flex-end' flexWrap='wrap' marginBottom={theme.spacing(3)}>
             <Typography fontSize={numberFontSize(totalPlants)} fontWeight={600} lineHeight={1}>
-              {totalPlants}
+              {totalPlants ? numericFormater.format(totalPlants) : ''}
```

Comment:
I don't think `totalPlants` can be undefined here, and this will display `''` for zero, which is not what we want

Reply:
I think this should be okay as `{numericFormater.format(totalPlants)}`